docs/military/darwin.md

```diff
@@ -4,4 +4,189 @@
+## Airspace
+### TCU
+
+DN TCU owns the airspace within a 40NM radius of the DN DME from `SFC`–`FL180`  
+
+DN TCU is also responsible for Active Restricted Area R264 A-K and R230 A-F  
+
+DAW may request DN TCU (`SFC`–`FL240`) from TRT to facilitate military transits to/from R264 A-K and R230 A-F  
+
+When both DN TCU positions are opened, DN TCU is split east and west along the runway 18/36 extended centreline. DAW owns the airspace to the west of the line and DAE owns the airspace to the east of the line  
+
+<figure markdown>
+![Default DN TCU Upper Limits](img/dntcu.png){ width="700" }
+  <figcaption>DN TCU Airspace</figcaption>
+</figure>
+
+### ADC
+
+ADC owns the airspace within the DN CTR (`SFC`–`A010`). This airspace is designed to facilitate the processing of helicopter scenic flights and low-level helicopter circuits.  
+The CTR extends 7NM from the thresholds of runways 11 and 29 but does not including the Robertson Barracks transit zone.  
+
+ADC may request DN TCU (`SFC`–`A020`) from DAW to facilitate fixed-wing circuit operations:  
+a) Fixed-wing circuit operations are typically conducted at Delissaville (YDLV) due to high traffic density at Darwin  
+b) ADC may deny requests for circuits if IFR traffic may be unduly delayed  
+
+<figure markdown>
+![WLM ADC Airspace](img/dnadc.png){ width="700" }
+  <figcaption>DN ADC Airspace</figcaption>
+</figure>
+
+### Surveillance
+DN ADC is permitted to use Surveillance standards for separation. Surveillance coverage can be expected to be available at all levels in the DN ADC airspace.  
+For simulation purposes, visual separation is assumed to exist below the cloud base, and within 7nm of the aerodrome. Visual separation can still be used to separate from aircraft on an instrument approach, below the cloud base.
+
+## Aerodrome
+### Standard Taxi Routes
+Aircraft transiting to or from the Southern and Northern GA should follow the following standardised taxi routes:  
+a) Outbound via V1  
+b) Inbound via Y1  
+
+Traffic permitting, aircraft above 136,000kg MTOW shall be entered/exited for RWY 11/29 from taxiways B2, C3, D or E2 and backtracked as required.  
+
+Taxiway V has a passing bay, located adjacent to the ARFF section. The passing bay is the preferred location for engine run-ups, no longer than 3 minutes  
+
+### Runway Modes
+
+Runway selection shall be coordinated with DAW. In any case, a change of runway mode shall not be broadcast on the ATIS until DAW accepts the change.  
+Darwin utilises Land and Hold Short Operations (LAHSO) to achieve an expeditious flow of traffic.
+
+If conditions are suitable, LAHSO runway configurations are preferred:
+
+a) Runway 29 and Runway 36 (LAHSO);  
+b) Runway 29 and Runway 18 (LAHSO);  
+c) Runway 29 only;  
+d) Runway 11 and Runway 36 (LAHSO);  
+e) Runway 11 and Runway 18;  
+f) Runway 11 only.  
+ 
+### Departures
```

Comment:
This can be removed cause it's already in coord section

Reply:
Remove What?